src/main/java/org/pesmypetcare/webservice/dao/PetDaoImpl.java

```diff
@@ -26,27 +26,45 @@
     private final String INVALID_PET_EXC;
     private CollectionReference usersRef;
 
+    private StorageDao storageDao;
+
     public PetDaoImpl() {
         Firestore db;
         db = FirebaseFactory.getInstance().getFirestore();
         usersRef = db.collection("users");
+        storageDao = new StorageDaoImpl(this);
 
         PETS_KEY = "pets";
         DELFAIL_KEY = "deletion-failed";
         PET_DOES_NOT_EXIST_EXC = "The pet does not exist";
         INVALID_PET_EXC = "invalid-pet";
     }
 
+    /**
+     * Gets the storage dao
```

Comment:
Missing a period at the end of the comment

Reply:
Not entirely my fault, Codacy didn't check it as an issue either.